pom.xml

```diff
@@ -133,6 +133,11 @@
             <version>4.5.10</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+          <groupId>org.apache.commons</groupId>
+          <artifactId>commons-lang3</artifactId>
+          <version>3.9</version>
```

Comment:
I'm not as familiar with the details of Java package/dependency management:

Will dependencies in the POM file impact installation of the plugin (or is this just for development)? Or are we using libraries that are standard on Jenkins installs?